### PR TITLE
Form 249/RJSF markdown in accordion

### PR DESCRIPTION
**Pull request by ValeriiMedvezhonkov**
@mpinter Is it the right way I am using markdown content in UI schema for fields?

This branch an PR only for testing

### CODE REVIEW

**Comment by mpinter** [comment]
@ValeriiMedvezhonkov yup :+1: , I think this makes sense, let's do it

**Comment by ValeriiMedvezhonkov** [comment]
@mpinter Waiting for approve Field-Group RJSF task, before finishing this task, pls review existing code

File: next/components/forms/widget-wrappers/WidgetWrapper.tsx
Comment (mpinter) [inline next/components/forms/widget-wrappers/WidgetWrapper.tsx:43]: instead of this use this approach (version for mapped elements) - https://bratislava.github.io/frontend/components#use-generated-ids

File: next/utils/utils.ts
Comment (mpinter) [inline next/utils/utils.ts:56]: I assume it's from here ? https://stackoverflow.com/questions/8511281/check-if-a-value-is-an-object-in-javascript if so (and in fact, even if it isn't) please add this link for context

```suggestion
// https://stackoverflow.com/questions/8511281/check-if-a-value-is-an-object-in-javascript
export const isObject = (value: any) =>
```

**Review by mpinter — APPROVED** [review]
Please fix the two comments before merging but otherwise good to go

**Pull request merged**
